tests/hypothesis_strategies/kernels.py

```diff
@@ -35,3 +47,18 @@ def kernels(draw: st.DrawFn):
         )
     else:
         return base_kernel
+
+
+@st.composite
+def kernels(draw: st.DrawFn):
+    """Generate :class:`baybe.kernels.basic.Kernel`."""
+    kernel_type = draw(st.sampled_from(KernelType))
+
+    if kernel_type is KernelType.SINGLE:
+        return draw(single_kernels())
+
+    base_kernels = draw(st.lists(single_kernels()))
+    if kernel_type is KernelType.ADDITIVE:
```

Comment:
If I understand this test correctly, this only tests `AdditiveKernels` that were built from `SimpleKernels` right? So there is no test that nests `AdditiveKernels` or `ProductKernels`?

Reply:
Yes, as answered via chat. I think this suffices for now and we can always extend if needed 👍🏼